sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -92,24 +90,6 @@ Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to creat
   az keyvault show --name <your-key-vault-name>
   ```
 
-## Key concepts
-
-- The **Certificates client** is the primary interface to interact with the API methods
-  related to certificates in the Azure Key Vault API from a JavaScript application.
-  Once initialized, it provides a basic set of methods that can be used to
-  create, read, update and delete certificates.
-- A **Certificate version** is a version of a certificate in the Key Vault.
-  Each time a user assigns a value to a unique certificate name, a new **version**
-  of that certificate is created. Retrieving a certificate by a name will always return
-  the latest value assigned, unless a specific version is provided to the
-  query.
-- **Soft delete** allows Key Vaults to support deletion and purging as two
-  separate steps, so deleted certificates are not immediately lost. This only happens if the Key Vault
-  has [soft-delete](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-ovw-soft-delete)
-  enabled.
-- A **Certificate backup** can be generated from any created certificate. These backups come as
-  binary data, and can only be used to regenerate a previously deleted certificate.
-
 ## Authenticating with Azure Active Directory
 
 The Key Vault service relies on Azure Active Directory to authenticate requests to its APIs. The [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) package provides a variety of credential types that your application can use to do this. The [README for `@azure/identity`](/sdk/identity/identity/README.md) provides more details and samples to get you started.
```

Comment:
I don't think that /sdk/identity/identity/README.md link will work in npm, right?

Reply:
It does work! try it yourself here: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-certificates#authenticating-with-azure-active-directory

Reply:
NPM/GitHub are smart about links that are relative to GitHub. They'll get converted automatically to the right base URL.

Reply:
Yeah, @sadasant just showed me. I love these tricks. :)